app/experimenter/settings.py

```diff
@@ -428,6 +428,21 @@
 GS_PROJECT_ID = "experiments-analysis"
 GS_BUCKET_NAME = "mozanalysis"
 
+# User uploads, e.g. branch screenshots
+UPLOADS_FILE_STORAGE = config(
+    "UPLOADS_FILE_STORAGE",
+    default=DEBUG
+    and "django.core.files.storage.FileSystemStorage"
+    or "storages.backends.gcloud.GoogleCloudStorage",
+)
```

Comment:
1) I wouldn't tie this to debug so we can support testing with any combination of local/gcp storage and debug on/off (ie using file storage locally with debug off to replicate prod)
2) I also wouldn't bother putting the defaults in here, just read the value from .env and require that it's in there and put the local one in .env.sample